AndroidSDKTests/src/test/java/com/leanplum/internal/RequestTest.java

```diff
@@ -67,6 +68,20 @@ public void setUp() {
     Leanplum.setApplicationContext(context);
   }
 
+  /**
+   * Test that read writes happened sequentially when calling sendNow().
+   */
+  @Test
+  public void testMultiThreaded() {
```

Comment:
test style: for test readability and maintainability it's better when test names are complete sentences:

public void shouldWriteAndReadInSequenceWhenRequestIsExecuted()

(something like that :) )